daemons/fenced/pacemaker-fenced.c

```diff
@@ -1377,15 +1377,216 @@ st_peer_update_callback(enum crm_status_type type, crm_node_t * node, const void
     }
 }
 
+static pcmk__cluster_option_t fenceder_options[] = {
```

Comment:
"fenced_options" or "fencer_options" would be a bit better

("fenced" comes from the daemon name pacemaker-fenced, while "fencer" describes its function)